train.py

```diff
@@ -0,0 +1,207 @@
+import argparse
+from typing import Any
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
```

Comment:
```
train.py|6 col 1 error| 'torch.nn.functional as F' imported but unused [F401]
```

I recommend running pre-commit hooks before every commit - you should get a warning about this and maybe black would clean it up for you. 

I don't know what's your setup in that regard, but when you set it up following the README, hooks should get triggered every time you run `git commit` in the terminal.

Reply:
I'll do that in next commit. I don't know why but they are triggering automatically when I do commit. I'll look into that.